eth/integrity/trie.go

```diff
@@ -38,6 +38,14 @@ func Trie(tx ethdb.Tx, slowChecks bool, quit <-chan struct{}) {
 			if errc != nil {
 				panic(errc)
 			}
+			select {
+			default:
```

Comment:
this default is kinda superfluous (although i see it was already there before PR )

Reply:
Without default loop will stuck until receive message from any channel - will do 1 iteration every 30 sec - instead of running non-stop and print 1 logs line every 30sec. 

Yes, this code existed before PR, but it was in end of loop - and loop has “early termination” (see variable slowChecks). It means with some combination of cli flags program didn’t print any logs.

Reply:
ah - you're right - I misread the default as a fallthrough